alphastats/gui/utils/options.py

```diff
@@ -5,130 +5,133 @@
 from alphastats.loader.SpectronautLoader import SpectronautLoader
 from alphastats.loader.GenericLoader import GenericLoader
 from alphastats.loader.mzTabLoader import mzTabLoader
+from alphastats.gui.utils.ui_helper import StateKeys
 
 
 def plotting_options(state):
+    dataset = state[StateKeys.DATASET]
```

Comment:
Should we maybe just pass dataset and metadata_columns directly instead of the whole state?

Reply:
for now this is fine I guess